packages/nimble-components/src/table-column/number-text/models/tests/decimal-formatter.spec.ts

```diff
@@ -2,81 +2,134 @@ import { getSpecTypeByNamedList } from '../../../../utilities/tests/parameterize
 import { DecimalFormatter } from '../decimal-formatter';
 
 describe('DecimalFormatter', () => {
+    const locales = ['en', 'de'] as const;
```

Comment:
`en` and `de` are somewhat arbitrary choices for locales to test with. I also could have expanded this list to include a bunch more locales (or even just the ones that NI traditionally translates to). However, I decided to keep things relatively simple and test with 2 locales. I think this gives us the coverage we want, which is ensuring that all our formatters are using the specified locale without adding a bunch of extra tests that are only testing that the `Intl.NumberFormat` behaves the way we expect for each locale.

Reply:
Even this seems like overkill. I would have left the test cases using one locale, and just added one extra test that changes the locale to make sure the formatted value adjusts accordingly. Still, I don't think it's a big deal to over-test a bit.

Reply:
I'll leave this discussion open so that owners can weigh in with their opinions. If no one has a strong opinion, I'll leave this as-is.

Reply:
I went back and forth while reviewing the tests but ended up deciding I like this level of coverage. A carefully-chosen single test would ensure that at least one code path in the formatter is respecting the locale but this approach increases our odds of proving that all code paths in the formatter are respecting the locale (important for the default formatter which uses multiple Intl.NumberFormats).